user_account/api/serializers.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework import serializers
+from user_account.models.user import UserAccount
```

Comment:
use some tool to sort imports.

for example:
https://medium.com/@cereblanco/setup-black-and-isort-in-vscode-514804590bf9

isort